src/ReactLeafletMap/index.js

```diff
@@ -234,16 +235,29 @@ class ReactLeafletMap extends React.PureComponent {
 
     getWmsTileLayer(layer, i) {
         const o = layer.options;
+        let layers = (o.params && o.params.layers) || '';
```

Comment:
Use optional chaining for better readability

Reply:
We do not have optional chaining package in this repo yet. But I can add it now.